### ISSUE TITLE
Is it possible to integrate `ajv-oai` in stead of `ajv` 

### ISSUE THREAD

**Issue by kaasdude** [anttiviljami/openapi-backend#39]
Hello,

I'm in the process of do an investigation on how we can migrate from swaggerize-express to your implementation for OpenAPI 3.

Got these warnings in formats as an example:
```
unknown format "int64" ignored in schema at path "#/properties/path/properties/id"
unknown format "int64" ignored in schema at path "#/properties/path/properties/id"
```

Your project uses `ajv` to validate the validation of JSON, where https://github.com/BiteBit/ajv-oai validates also on the additional OpenAPI formats.

I'm not sure if this is a drop in replacement, but is it possible to integrate `ajv-oai` in stead of `ajv` ?

**Comment by anttiviljami** [comment]
Hi @kaasdude!

I've just released openapi-backend`@2.3.1`, which includes the [customizeAjv](https://github.com/anttiviljami/openapi-backend/blob/master/DOCS.md#parameter-optscustomizeajvoriginalajv-ajvopts-validationcontext) option.

You can try substituting the Ajv validator with yours. Like so:

```javascript
const api = new OpenAPIBackend({
  customizeAjv: (originalAjv, ajvOpts) => {
    return new AjvOAI();
  },
  // ... your definition here
});
```

Please report back how it goes :)

**Comment by kaasdude** [comment]
Seems to just work!

**Got this:**

```yaml
  '/pets/{id}':
    get:
      summary: Get pet by its id
      parameters:
        - name: 'id'
          in: path
          required: true
          description: Yadadada.
          schema:
            type: integer
            format: int64
            minimum: 1
      responses:
        '200':
          description: OK
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Pet'
```

**And this:**

```javascript
        // Enable OpenAPIBackend for incoming requests.
        const api = new OpenAPIBackend({
            customizeAjv: (originalAjv, ajvOpts) => new AjvOAI({
                nullable: true,
                unknownFormats: true,
                schemaId: 'auto'
            }),
            definition: ...,
            handlers: ...
        });
```

I do need to test if I exceed the memory of for example an `int32` formatted integer.
